haas/ext/auth/database.py

```diff
@@ -131,6 +131,17 @@ def user_remove_project(user, project):
     db.session.commit()
 
 
+@rest_call('PUT', '/auth/basic/user/<user>/set_admin', Schema({
+    'user': basestring,
+    'is_admin': bool,
+}))
+def user_set_admin(user, is_admin):
+    get_auth_backend().require_admin()
```

Comment:
Would like to see some mechanism to prevent someone from removing themselves from admin.